File: javascript/ex7.js
/*

7. Remova duplicatas
Dado um array com números repetidos, retorne um novo array apenas com
valores únicos.

*/

const array = require("../array.json")

let arrayLimpo = []

array.forEach(num => {
    num = parseInt(num)

    if (!arrayLimpo.includes(num)){
        arrayLimpo.push(num)
    }
})

console.log("Array limpo: " + arrayLimpo)

File: python/ex7.py
# 7. Remova duplicatas
# Dado um array com números repetidos, retorne um novo array apenas com
# valores únicos.

from utils import array

array_unico = []
array_unico.append(array[0])

for num in array:
    if num not in array_unico:
        array_unico.append(num)

print(array_unico)